Content.Server/Morgue/Components/MorgueEntityStorageComponent.cs

```diff
@@ -94,7 +95,10 @@ protected override void OpenStorage()
                 _tray = Owner.EntityManager.SpawnEntity(_trayPrototypeId, Owner.Transform.Coordinates);
                 var trayComp = _tray.EnsureComponent<MorgueTrayComponent>();
                 trayComp.Morgue = Owner;
-                EntityQuery = new IntersectingEntityQuery(_tray);
+                var physicsManager = EntitySystem.Get<SharedBroadPhaseSystem>();
+                var box = new Box2(Owner.Transform.Coordinates.Position + new Vector2(-0.2f, -0.2f),
+                                   Owner.Transform.Coordinates.Position + new Vector2(-0.2f, -0.2f));
+                CollidingEntities = physicsManager.GetCollidingEntities(_tray.Transform.MapID, in box);
```

Comment:
this is ugly but I'm not sure how to check for collisions better. But it looked like it worked when I tested locally.

Reply:
bit better now with WorldPosition